syntax = "proto3";
import "Common.proto";

// 请求登录 服务器登录验证
message Login_LoginReq   
{
	bytes login_name = 1;
	bytes login_pwd = 2;
	bytes game_id = 3;
	bytes open_id = 4;
	bytes token = 5;
	bytes channel_id = 6;
	bytes snid = 7;
	bytes mac = 8;
}

message Login_LoginReply
{
	int32 ret = 1;
}

message Login_CreateRoleReq
{
	int32 role_id = 1;
	bytes nick_name = 2;
	int32 sex = 3;
}

message Login_CreateRoleReply
{
	int32 ret = 1;
}

message Login_SelectRoleReq
{
	int32 role_id = 1;
}

message Login_SelectRoleReply
{
	int32 ret = 1;
}

message Login_DelRoleReq
{
	int32 role_id = 1;
}

message Login_DelRoleReply
{
	int32 ret = 1;
}